ample/util/mrbump_util.py

```diff
@@ -186,18 +198,27 @@ def _extractPurged(self, mrbump_dir):
                 purged_results[d['ensemble_name']] = d
         return purged_results
 
-    def extractResults(self, mrbump_dir, purge=False):
+    def extractResults(self, mrbump_dir, purge=False, debugPrint=False):
```

Comment:
camelCase 🤦‍♂️ 😑

Reply:
I know - the thing is it's already a mess of CamelCase and snake_case and should be updated to be all snake_case, but till then I thought it best to at least be consistent within the file?

Reply:
Maybe just update the file as and when you see it? I think Pycharm allows you to update the function names throughout your code

Reply:
I totally agree. just because it's already a mess doesn't mean it should be propagated. also, it's hardly consistent when another argument in the same function definition uses snake_case